AudioRenderCAPI/include/AudioRenderCAPI.h

```diff
@@ -0,0 +1,66 @@
+#ifndef AUDIO_RENDER_C_API_H
+#define AUDIO_RENDER_C_API_H
+
+#include <stdint.h>
+
+#if defined __cplusplus
+extern "C" {
+#endif
+
+typedef int32_t audioRender_Bool;
+typedef void audioRender_DrawDevice;
+
+struct audioRender_Rectangle {
+    float left;
+    float top;
+    float right;
+    float bottom;
+};
+
+struct audioRender_Point {
+    float x;
+    float y;
+};
+
+// Create draw device which uses audio as output
+__declspec(dllexport) audioRender_DrawDevice* audioRender_DeviceInitAudioRender(float scaleX, float scaleY);
```

Comment:
You should use a macro here that changes meaning when compiling the DLL or linking it into an app. The COMPILING_DLL can be defined e.g. in CMakeList

#if COMPILING_DLL
    #define DLLEXPORT __declspec(dllexport)
#else
    #define DLLEXPORT __declspec(dllimport)
#endif